Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -37,8 +37,6 @@ param (
         }
     }
```

Comment:
ensure we do not import any file /module from agent. We should import it from task folder only